code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -8,6 +8,7 @@
 	reload_delay = 30
 	ammo_mag = "box"
 	matter = list(MATERIAL_CARDBOARD = 1)
+	bad_type = /obj/item/ammo_magazine/ammobox
```

Comment:
What's this change for?

Reply:
The change to cheaper ammoboxes would spawn an ammobox with an empty icon state